src/Icons/icons/arrow-mark.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 32 20" fill="#4C4C4C" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M32 10L18 0v5H0v10h18v5l14-10z"/></svg>
```

Comment:
The viewbox is wrong here, it should be a square so 32 x 32. I can show you how to re-export it is you like, but basically export the one names "Drawing library icons" in Figma

Reply:
ahhhh I exported union